transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -405,6 +406,11 @@ public final ChannelPromise voidPromise() {
         return pipeline.voidPromise();
     }
 
+    final void autoFlushModified(boolean autoFlush) {
```

Comment:
@NiteshKant I wonder if we need to use `synchronized` here so we not get into trouble in terms of ordering as someone could change it from multiple threads.